code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -1,5 +1,12 @@
 /obj/item/projectile/bullet/c9mm
-	damage = 18
+	damage = 22
+	sharp = 0
+	can_ricochet = TRUE
+
+/obj/item/projectile/bullet/c9mmh
+	damage = 1.2*22
```

Comment:
having math in here like this is wierd, best to just precalculate it and write it in.  26.4

Reply:
That's for future, so people know how these numbers got calculated. Does it look so weird?

Reply:
yea this is a bad method

If you want to make it based on the base class, then make the HV bullets a subclass, and have them modify the values in their New proc
and also make 1.2 a define